### PR TITLE
MC-175: Prevent bug during product creation if sku is a number.

### PR DESCRIPTION
**Pull request by damien-carcel**
Bug fix: [yes]
Feature addition: [no]
Backwards compatibility break: [no]
Phpspec specification passes: [no]
Checkstyle issues: [no]*
ChangeLog updated: [yes]
Fixes the following issue:

### CODE REVIEW

File: Webservice/Webservice.php [hunk 1]
```diff
@@ -891,10 +891,11 @@ protected function removeNonUpdatePart(array $productPart)
      */
     protected function updateProductIfMultipleStoreView(array $productPart)
     {
-        $storeViews = $this->getStoreViewsList();
-
-        if (count($storeViews) > 1 && count($productPart) === static::CREATE_PRODUCT_SIZE) {
-            $updateProductPart = array_slice($productPart, static::MAGENTO_STATUS_DISABLE);
+        if (count($this->magentoStoreViewList) > 1 && count($productPart) === static::CREATE_PRODUCT_SIZE) {
```
Comment (fitn) [inline Webservice/Webservice.php]: You've to call magentoStoreViewList with the getter to prevent count(null) ;)

File: Webservice/Webservice.php [hunk 2]
```diff
@@ -891,10 +891,11 @@ protected function removeNonUpdatePart(array $productPart)
...
+            $updateProductPart = array_merge(
+                array_slice($productPart, static::MAGENTO_STATUS_DISABLE),
+                ['sku']
```
Comment (fitn) [inline Webservice/Webservice.php]: "sku" is a header to send in Magento?

Comment (damien-carcel) [inline Webservice/Webservice.php]: It's a identifier that says to Magento: "I update the product identified by its sku, not its id!".

Comment (fitn) [inline Webservice/Webservice.php]: :+1:

**Pull request merged**